app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+
+    if parts[0] != "mv" or len(parts) != 3:
```

Comment:
The command validation checks if the first part is 'mv' and if there are exactly three parts. This is correct, but ensure that the command is always in the expected format before calling this function.